.github/workflows/main.yml

```diff
@@ -44,6 +44,12 @@ jobs:
       - name: ▶️ Run Jest Screenshot
         run: npm run jest-screenshot
 
+      - name: ▶️ Run Percy Screenshot
+        if: ${{ github.actor != 'renovate[bot]' }}
```

Comment:
Nice work figuring out how to stop renovate from using all of our budget for Percy screenshots 💯